flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -313,18 +313,12 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
             options.dist = dist
         }
 
-        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool,
-           enableAutoNativeBreadcrumbs == false {
-            options.integrations = options.integrations?.filter { (name) -> Bool in
-                name != "SentryAutoBreadcrumbTrackingIntegration"
-            }
+        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool {
+            options.enableAutoBreadcrumbTracking = false
         }
 
-        if let enableNativeCrashHandling = arguments["enableNativeCrashHandling"] as? Bool,
-           enableNativeCrashHandling == false {
-            options.integrations = options.integrations?.filter { (name) -> Bool in
-                name != "SentryCrashIntegration"
-            }
+        if let enableNativeCrashHandling = arguments["enableNativeCrashHandling"] as? Bool {
+            options.enableCrashHandler = false
```

Comment:
@brustolin can you confirm that the `enableCrashHandler` flag is the same as filtering `SentryCrashIntegration`? this flag didn't exist before.